app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffect.kt

```diff
@@ -4,10 +4,8 @@ import org.simple.clinic.settings.Language
 
 sealed class ChangeLanguageEffect
 
-object LoadCurrentSelectedLanguageEffect: ChangeLanguageEffect()
+object LoadCurrentSelectedLanguageEffect : ChangeLanguageEffect()
 
-object LoadSupportedLanguagesEffect: ChangeLanguageEffect()
+object LoadSupportedLanguagesEffect : ChangeLanguageEffect()
 
-data class UpdateSelectedLanguageEffect(val newLanguage: Language): ChangeLanguageEffect()
-
-object RestartActivityEffect: ChangeLanguageEffect()
```

Comment:
Curious, do we not need this anymore?

Reply:
I was planning to move the activity restart so that it's handled in the previous screen, if the current language has changed. But now I realize that it is exposing details of what happened in this screen in the previous screen. Can we pair on this part in a bit?

Reply:
Discussed offline.